components/list/list-item-checkbox-mixin.js

```diff
@@ -55,7 +62,7 @@ export const ListItemCheckboxMixin = superclass => class extends SkeletonMixin(s
 
 	constructor() {
 		super();
-		this.disabled = false;
+		this.selectionDisabled = false;
```

Comment:
Nit: This should also be placed alphabetically with respect to the other properties